Source/Core/Core/IOS/IOSC.cpp

```diff
@@ -420,9 +420,9 @@ static CertECC MakeBlankEccCert(const std::string& issuer, const std::string& na
 {
   CertECC cert{};
   cert.signature.type = SignatureType(Common::swap32(u32(SignatureType::ECC)));
-  std::strncpy(cert.signature.issuer, issuer.c_str(), 0x40);
+  memcpy(cert.signature.issuer, issuer.c_str(), 0x40);
```

Comment:
These are copying into a string buffer. strncpy is appropriate here. This doesn't actually resolve any potential warning and just hides it instead. This isn't adequate at all.

Reply:
I believe signature issuer is not a NULL-terminated string. am I right?

Reply:
you are right. I will fix.